lms/djangoapps/experiments/flags.py

```diff
@@ -178,4 +178,8 @@ def override(self, active=True, bucket=1):  # pylint: disable=arguments-differ
         if not active:
             bucket = 0
         with patch.object(self, 'get_bucket', return_value=bucket):
-            yield
+            # TODO We can move this import to the top of the file once this code is
+            # not all contained within the __init__ module.
+            from openedx.core.djangoapps.waffle_utils.testutils import override_waffle_flag
+            with override_waffle_flag(self, active):
+                yield
```

Comment:
Why was this bit needed?

Reply:
oh. For is_experiment_on

Reply:
Might be worth a comment to that effect? And maybe a test in test_flags.py?

Reply:
Good call